### ISSUE TITLE
Nonconvergence / texture-copy on custom dataset? 

### ISSUE THREAD

**Issue by mattbev** [nianticlabs/monodepth2#325]
My monodepth results have an artifact where the texture of the input RGB frame copies to the depth map. I saw [some posts](https://github.com/nianticlabs/monodepth2/issues/300#issue-778060523) indicating that this could be a nonconvergence issue or an issue with my intrinsics matrix, but I decreased the finetuning learning rate to 1e-5 with no effect and am pretty sure the intrinsics are correct (see bottom to check). Some examples are below:

**Input**: 224x128 crop, trained from scratch w/ default learning rate. The effect is more pronounced when finetuning.
![0000000001](https://user-images.githubusercontent.com/26170087/111721876-23cabc80-8837-11eb-89d3-ada487bd7dfd.jpg)

**Depth map**:
![scratch_224x128_disp](https://user-images.githubusercontent.com/26170087/111721788-fc73ef80-8836-11eb-8f53-91e30806b710.jpeg)

Here's how I define my intrinsics matrix:
```python
self.full_res_shape = (1920, 1208)
cx = 982.06345972
cy = 617.04862261
fx = 942.21001829
fy = 942.91483399
w, h = self.full_res_shape
self.K = np.array([[fx/w, 0,    cx/w, 0],
                   [0,    fy/h, cy/h, 0],
                   [0,    0,    1,    0],
                   [0,    0,    0,    1]], dtype=np.float32)
```

For what it's worth, I notice this issue is more pronounced as the size of my dataset is increased. Thanks for any help!

**Comment by mrharicot** [comment]
Hi,

It is difficult to know what is wrong here.
The intrinsics seem fine at a glance, however the input image has two issues with static parts at the top and bottom which tend to be problematic for training.
The input image size of 224x128 is also very small:  I would increase it to at least double.
Finally are you training in monocular mode? If so what is the framerate of the input video? if it is too high or too low it will be an issue for convergence.

I hope this helps!
